src/test/resources/log4j2-test.xml

```diff
@@ -17,7 +17,7 @@
 <Configuration status="WARN">
   <Appenders>
     <Console name="STDOUT" target="SYSTEM_OUT">
-      <PatternLayout pattern="[%d] [%t] [%c] %p %m%n" />
+      <PatternLayout pattern="[%d] [%t] [%c] %p [%X{traceId:-},%X{spanId:-}] %m%n" />
```

Comment:
I don't see anything in logs though.
Do I need to have something else for MDC to be populated automatically ?
For example some special `ObsrvationHandler`...

Reply:
that should work oob

Reply:
Maybe it works with Spring Boot, but here I use a plain Brave and `ObservationRegistry` in the unit test and I have a Log4J and slf4J as deps for logging.
See `ReactorKafkaObservationTests` for more details.

Reply:
It just doesn't layout a simple observation in logs like this:
```
Observation.createNotStarted("test observation", OBSERVATION_REGISTRY)
                .observe(() -> log.trace("Current trace"));
```
🤷

Reply:
So, turns out I need to do this in my unit test:
```
        ThreadLocalCurrentTraceContext traceContext =
                ThreadLocalCurrentTraceContext.newBuilder()
                        .addScopeDecorator(MDCScopeDecorator.get())
                        .build();
```
Instead of plain `ThreadLocalCurrentTraceContext.create()`.
And now I can see a log correlation for my observations.